<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogR</class>
 <widget class="QDialog" name="DialogR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>518</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>719</width>
    <height>518</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>719</width>
    <height>518</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-image: url(:/new/prefix1/resourse/login/2.jpg);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>471</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 24pt &quot;微软雅黑 Light&quot;;</string>
   </property>
   <property name="text">
    <string>30s快捷注册 LetsGo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border-image:transparent;
font: 75 16pt &quot;微软雅黑&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;微软雅黑&quot;;
background: transparent;
border-image:transparent;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>口令</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>541</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;微软雅黑&quot;;
background: transparent
</string>
   </property>
   <property name="title">
    <string>性别</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>男生</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;微软雅黑&quot;;
background: transparent</string>
    </property>
    <property name="text">
     <string>女生</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
font: 75 16pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>年龄</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>340</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
font: 75 16pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>340</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
background: transparent</string>
   </property>
   <property name="text">
    <string>地区</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>420</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
font: 75 16pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>420</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>职业</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>420</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
font: 75 16pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>420</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;微软雅黑&quot;;
background: transparent</string>
   </property>
   <property name="text">
    <string>电话</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>460</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
color: rgb(255, 255, 170);
font: 75 20pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>开始畅游</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
